py_gtfs_rt_ingestion/lib/convert_gtfs_rt.py

```diff
@@ -263,13 +331,13 @@ def move_s3_files(self) -> None:
         move archive and error files to their respective s3 buckets.
         """
         if len(self.error_files) > 0:
-            move_s3_objects(
+            self.error_files = move_s3_objects(
```

Comment:
is there a reason to set `self.error_files` and `self.archive_files` to the return here?

Reply:
i guess we're going to try and move them _the next time_ we write a table? but that doesn't feel like the right way to handle this to me.

Reply:
It's lazy retry logic. `move_s3_objects` returns a list of any paths that weren't moved. So it will almost always return an empty list, but in the case something doesn't get moved, it could retry on the next call to `self.move_s3_files`